content/03.methods.md

```diff
@@ -6,7 +6,14 @@ The Pediatric Brain Tumor Atlas specimens are comprised of samples from Children
 Blood and tumor biospecimens from patients enrolled within the CBTTC were sent to the Children's Hospital of Philadelphia for RNA and/or DNA extraction.
 PNOC collected blood and tumor biospecimens from newly-diagnosed DIPG patients as part of the clinical trial [PNOC003/NCT02274987](https://clinicaltrials.gov/ct2/show/NCT02274987) [@doi:10.1002/ijc.32258]. 
 
-#### Nucleic acids extraction
+#### Nucleic acids extraction and library preparation
+
+##### PNOC samples
+Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed DNA and RNA extractions on tumor biopsies using the Qiagen AllPrep Kit. 
+All RNAs used for library prep had a minimum RIN of 7 but no QC thresholds were implemented for the DNAs. 
+For library preparation, 500ng of nucleic acids were used as input for RNA-Seq and WXS.
```

Comment:
Is WXS for whole-exome sequencing? Is this already defined elsewhere?

Reply:
> This is not terribly detailed. Would someone be able to follow these steps to end up with the same process? Did they specifically follow the Qiagen AllPrep Kit manufacturer's instructions? If so, we should say that and ideally link the instructions/protocol here.

I know. I have another email out to them asking for more specifics, but wanted to get in what I had so far. Mateusz will also be working on this section for CBTTC samples soon (today?), so wanted to get some of the details in so he can follow.

Reply:
> Is WXS for whole-exome sequencing? Is this already defined elsewhere?

Yes, and I assumed we will be defining these abbreviations in the intro to describe the dataset, so did not do it here (unless you think each section will need this spelled out).